label_studio/core/label_config.py

```diff
@@ -76,22 +77,37 @@ def _fix_choices(config):
     return config
 
 
-def parse_config_to_json(config_string):
+def parse_config_to_xml(config_string: Union[str, None], raise_on_empty: bool = False) -> Union[OrderedDict, None]:
+    if config_string is None:
+        if raise_on_empty:
+            raise TypeError('config_string is None')
+        return None
+
+    xml = etree.fromstring(config_string, forbid_dtd=True)
+
+    # Remove comments
+    for comment in xml.findall('.//comment'):
+        comment.getparent().remove(comment)
+
+    return xml
+
+
+def parse_config_to_json(config_string: Union[str, None]) -> Tuple[Union[OrderedDict, None], Union[str, None]]:
     try:
-        xml = etree.fromstring(config_string, forbid_dtd=False)
+        xml = parse_config_to_xml(config_string, raise_on_empty=True)
     except TypeError:
         raise etree.ParseError('can only parse strings')
     if xml is None:
         raise etree.ParseError('xml is empty or incorrect')
     config = xmljson.badgerfish.data(xml)
     config = _fix_choices(config)
-    return config
+    return config, etree.tostring(xml)
```

Comment:
This is still required, otherwise the encoding used by default in the lib causes:

`cannot use a string pattern on a bytes-like object` as the value returned is a bytestring.

```suggestion
    return config, etree.tostring(xml, encoding='unicode')
```